api.py

```diff
@@ -167,6 +167,44 @@ def list_group_members(team_id, group_id, **kwargs):
     return make_get_request('team/{}/group/{}/users'.format(team_id, group_id), **kwargs)
 
 
+def set_owner(team_id, resource_id, **kwargs):
+    return make_post_request({}, 'acl/{}/transfer-to-team/{}'.format(resource_id, team_id), **kwargs)
+
+
+def list_permissions(resource_id, **kwargs):
+    return make_get_request('acl/{}/view'.format(resource_id), **kwargs)
+
+
+def set_public_permissions(resource_id, read, write, share, **kwargs):
```

Comment:
Unrelated so hopefully a different "conversation".  Should we have share permissions?  Seems like this feature is basically going to be unused.